content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,220 @@ We have deprecated the following cert-manager APIs:
+running as non-root:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    runAsNonRoot: true
+```
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you are using custom containers that run as root with the Helm chart, you
+will need to set this back to `false`.
+
+{{% /pageinfo %}}
+
+## CA Issuer and `ca.crt`
+
+The CA issuer [now](https://github.com/jetstack/cert-manager/pull/3865) attempts
+to store the root CA instead of the issuing CA into the `ca.crt` field for
+issued certificates. All of the information which was previously available is
+still available: the intermediate should appear as part of the chain in
+`tls.crt`.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  This is a change of behavior with the CA Issuer, make sure that the change in
+`ca.crt` does not impact you current use.
+
+{{% /pageinfo %}}
+
+# New Features
+
+## Support for the built-in CertificateSigningRequests
+
+It is [now](https://github.com/jetstack/cert-manager/pull/4064) possible to use
+the built-in Kubernetes [CertificateSigningRequest][] resources with
+cert-manager. The CA Issuer is currently the only supported issuer. The feature
+is experimental and can be enabled by adding a flag to the cert-manager
+controller. For example, with Helm:
+
+```sh
+helm install cert-manager jetstack/cert-manager \
+  --set extraArgs="{--feature-gates=ExperimentalCertificateSigningRequestControllers=true}"
+```
+
+Imagining that you have an existing CA issuer named `my-ca-issuer` in the
+default namespace. To let cert-manager sign your CSR, you (the entity creating
+the CSR) need a [specific
+role](../../usage/kube-csr/#referencing-namespaced-issuers). You can then
+create a CertificateSigningRequest:
+
+```sh
+openssl genrsa -out test.key 2048
+kubectl apply -f - <<EOF
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+metadata:
+  name: test
+spec:
+  groups:
+  - system:authenticated
+  request: $(openssl req -new -key test.key -out /dev/stdout | base64 | tr -d "\n")
+  signerName: issuers.cert-manager.io/default.my-ca-issuer
+  usages:
+  - client auth
+EOF
+```
+
+You will still need to manually approve the CSR since cert-manager does not
+approve CSRs by default
+([unlike](../../concepts/certificaterequest/#approval)
+CertificateRequests that get automatically approved by default):
+
+```sh
+kubectl certificate approve test
+```
+
+cert-manager then signs the CSR. To know more about it, see [the
+documentation](../../usage/kube-csr) on how use the built-in
+CertificateSigningRequests with cert-manager.
+
+[CertificateSigningRequest]: https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/
+
+## Vault Issuer handle of `ca.crt`
+
+The Vault issuer is now able to fill the `ca.crt` in Secrets. To do that,
+cert-manager reconstructs the certificate chain that is returned by Vault and
+uses the certificate at the start of the chain as the `ca.crt`.
+
```

Comment:
"start of the chain" is ambiguous. 
If we are going to explain it, it should be in the docs for the Vault issuer.

Reply:
I think it is not worth detailing this, so I removed the mention to "start of the chain" in a3d6a65.